lldb/include/lldb/Target/StackFrameList.h

```diff
@@ -49,6 +49,14 @@ class StackFrameList {
   /// Resets the selected frame index of this object.
   void ClearSelectedFrameIndex();
 
+  /// Return \code true if the next frame is hidden. False otherwise or if it's
+  /// the last frame.
```

Comment:
```suggestion
  /// Return \code true if the next frame is hidden.
```

I'd say it's implied that if we're at the first/last frame, then querying anything about the previous/next frame is not going to be valid. Not sure we need to document that